content/work/adobe.md

```diff
@@ -29,6 +52,40 @@ press:
       colleagues.
     <<: *meghan
     slug: fun
+  - text: |
+      Oddbird developers know what they’re doing. 
+      We appreciate you all driving us to our goals.
+      **I’m glad usability testing was baked into the process.
+      It really was so valuable.** We are super super happy.
+    <<: *meghan
+    slug: usability
+planning:
+  - title: Game Goals
+    icon: icons/map
+    text: |
+      We started with a conversation about goals for the game — create a happy place for designers to connect with fonts and a method for Adobe to collect their feedback. We talked about what the game should not do like sell things, store player information, or become a social media platform. We defined technical and accessibility requirements, collected specifics about the content, and established a deadline. 
+  - title: Object Map
+    icon: icons/site-map
+    text: |
+      Using Sophia Prater’s Object Oriented UX strategy, we mapped out the objects in the web application. Though this project had only four objects, the map helped us agree on terms and get organized. We fleshed out the content of each object and the relationships between the four objects that would make up the game: question, option, superpower, and session user.
+  - title: Interactive Wireframe
+    icon: icons/interact
+    text: |
+      We sketched a wireframe of the game showing the flow of a player through the objects and ways to navigate from one object to another. Illustrations and animations make up the bulk of this game, but we didn’t get into those details yet at this stage. Keeping the wireframe basic and grayscale helped us nail down when animations should occur, what illustrations we would need, and where important Calls to Action should be highlighted throughout the game. 
+  - title: User Profiles
+    icon: icons/users
+    text: |
+      For the user profile, we described the game play of a typical player. Thinking through where the player may be coming from — an Adobe promo or a social media post — helped us hone our ideas for the landing page. In order to make the game accessible to as many players as possible, we establish that both sound and animation should start in the off position, with prominent toggles to turn them on at the player’s discretion.
+  - title: User Stories
+    icon: icons/theater
+    text: |
+      Using Goals, Object Map, Wireframe, and User Profiles we will created a list of expected features or ‘stories’ written from the perspective of an individual player.
+      Each user story defined one task that a player might want to accomplish during the game. (As a Player, I want to select a font option.)
+  - title: Itemized & Prioritized Estimate
```

Comment:
@SondraE can you look at this as well?

Reply:
@mirisuzanne I opted to add Adobe-specific info instead of removing that. Thoughts?

  - title: User Stories
    text: |
      Using the Goals, Object Map, Wireframe, and User Profiles 
      we created a list of expected features or ‘stories.’ 
      Each user story defined one task 
      that a player might want to accomplish 
      during the game. 
      For example, _As a Player, I want to select a font 
      option._  Adobe appreciated this outline of the project.
      It help our big team,
      which included people from multiple companies, 
      stay organized and on task –
      focused on the goals.
  - title: Itemized & Prioritized Estimate
    text: |
      We estimated each user story, 
      providing Adobe with an itemized hourly estimate, 
      organized into phases.
      Structuring the estimate in this way
      allowed Adobe to adjust priorities,
      such as limiting the typographic superpowers to five
      for the initial release.